build_msvc/README.md

```diff
@@ -57,19 +57,13 @@ PS >py -3 msvc-autogen.py
 
 - An optional step is to adjust the settings in the `build_msvc` directory and the `common.init.vcxproj` file. This project file contains settings that are common to all projects such as the runtime library version and target Windows SDK version. The Qt directories can also be set.
 
-- To build from the command line with the Visual Studio 2017 toolchain use:
-
-```
-msbuild /m bitcoin.sln /p:Platform=x64 /p:Configuration=Release /p:PlatformToolset=v141 /t:build
-```
-
 - To build from the command line with the Visual Studio 2019 toolchain use:
 
 ```
 msbuild /m bitcoin.sln /p:Platform=x64 /p:Configuration=Release /t:build
 ```
 
-- Alternatively open the `build_msvc/bitcoin.sln` file in Visual Studio.
```

Comment:
If we don't have a note that < Visual Studio 2019 is not supported, won't this lead to people opening issues that they can't build with an earlier version?

Reply:
I believe the leveldb appveyor file and readme are from the upstream repo. EIther way they are not used in the msvc CI for Bitcoin Core.

There is no Visual Studio 2016 application. Instead to make things super confusing Microsoft have used version 16.x for Visual Studio 2019, see https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#History.

That's a good point about the build not working with earlier versions of Visual Studio. I'll adjust the PR to add a note to that effect.